SparkPost/2web2ui
# SA-2247: Remove most of inbox placement pages

### What Changed

- Removed inbox placement pages
- Separated seedlist page
- Moved seedlist page to `Configuration` top level nav

### How To Test

- Go to an account
- Verify inbox placement nav item no longer exists
- Verify that the Seedlist page is under the configuration nav item
- Verify that the seedlist page still is under the inbox placement feature flag

### To Do
